gtest_parallel.py

```diff
@@ -454,7 +454,7 @@ def find_tests(binaries, additional_args, options, times):
 
       test_count += 1
 
-  return tasks
+  return sorted(tasks)
```

Comment:
I'll do a follow-up change with this separately once I've added tests for and submitted this first. :)